src/PixiEditor/Views/MainWindow.xaml

```diff
@@ -178,13 +178,13 @@
                             </MenuItem.ItemTemplate>
                         </MenuItem>
                         <MenuItem
-                            Header="_Save"
+                            Header="_Save (.pixi)"
                             cmds:Menu.Command="PixiEditor.File.Save" />
                         <MenuItem
-                            Header="_Save As..."
+                            Header="_Save As... (.pixi)"
                             cmds:Menu.Command="PixiEditor.File.SaveAsNew" />
                         <MenuItem
-                            Header="_Export"
+                            Header="_Export (.png, .jpeg, .gif, .bmp)"
```

Comment:
This is a bit too long, I'd replace it with "_Export (.png etc.)"

Reply:
".png etc." sounds broken, maybe ".png, .jpg, etc."

Reply:
ok, ".png, .jpg, etc" then